apps/public-docsite-v9/src/Concepts/Theming.stories.mdx

```diff
@@ -145,3 +145,34 @@ import { themeToTokensObject } from '@fluentui/react-components';
 
 export const customTokens = themeToTokensObject(customLightTheme);
 ```
+
+### Creating theme styles without `FluentProvider`
```

Comment:
I don't think that this approach scales: Storybook does not have proper search and this is hidden in depth 🐱 

Can we add a separate section there? Then we can can add proper examples:

<img width="236" alt="image" src="https://github.com/microsoft/fluentui/assets/14183168/44aad762-a8c8-4b0e-8763-560b720664d1">